client/landing/stepper/declarative-flow/internals/steps-repository/setup-blog/index.tsx

```diff
@@ -37,7 +37,7 @@ const SetupBlog: Step = ( { navigation, flow } ) => {
 
 	useEffect( () => {
 		// Clear site title and show placeholder for the flows below
-		if ( isStartWritingFlow( flow ) && siteTitle === 'Site Title' ) {
+		if ( siteTitle === 'Site Title' ) {
```

Comment:
Yeah, it also applies to other flows. My idea was to reuse it since we have a design approval and seems good to have in it all cases. How do you see?

Removed [the comment here](https://github.com/Automattic/wp-calypso/pull/77222/commits/fb3ce06b5fffb5d2cf9c181171c220dc5ac26719).